src/Harald.WebApi/Domain/Events/CapabilityDeletedDomainEvent.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using Newtonsoft.Json.Linq;
+
+namespace Harald.WebApi.Domain.Events
+{
+    public class CapabilityDeletedDomainEvent : IDomainEvent<CapabilityDeletedData>
+    {
+        public CapabilityDeletedDomainEvent(ExternalEvent domainEvent)
```

Comment:
We should really call this ExternalEvent by its true name, IntegrationEvent ;)

Reply:
`Code changed to follow recommendations` & IntegrationEvent does not inherit from IDomainEvent<object>